src/docs/product/issues/suspect-commits/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Suspect Commits
+sidebar_order: 2
+redirect_from:
+  - /product/releases/suspect-commits/
+description: "Learn how integrations enable suspect commits."
+---
+
+## Suspect Commits & Suggested Assignees
```

Comment:
The text after this heading introduces the idea of why you'd use suspect commits. Adding a heading here makes it feel like we've missed something that we needed to know before we started learning about suspect commits & suggested assignees, so we should take this out.

```suggestion
```